extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -14,23 +14,112 @@
  * limitations under the License.
  */
 
+.i-amphtml-story-interactive-slider-container {
+  --percentage: 0%;
+}
+
 .i-amphtml-story-interactive-slider-input-container {
+  display: flex !important;
   background: var(--i-amphtml-interactive-option-background-color) !important;
   border-radius: 2em !important;
   box-shadow: var(--i-amphtml-interactive-component-shadow) !important;
 }
 
+.i-amphtml-story-interactive-slider-bubble {
+  position: absolute !important;
+  transform: translate(-50%, -50%) scale(0);
+  width: 2.8em;
+  height: 2.8em;
+  top: 50%;
+  left: var(--percentage);
+  text-align: center;
+  line-height: 2.6em;
```

Comment:
Does it not work better if you match `line-height: 2.8em` like the height value?